File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseMapInfoFeaturesPublicTransport.scala

package org.openapitools.client.model

import java.time.OffsetDateTime

case class ResponseMapInfoFeaturesPublicTransport (
    _dateStart: OffsetDateTime,
    _dateEnd: OffsetDateTime
)
object ResponseMapInfoFeaturesPublicTransport {
    def toStringBody(var_dateStart: Object, var_dateEnd: Object) =
        s"""
        | {
        | "dateStart":$var_dateStart,"dateEnd":$var_dateEnd
        | }
        """.stripMargin
}

File: kotlin-spring/src/main/kotlin/org/openapitools/model/ResponseMapInfoFeaturesPublicTransport.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param dateStart 
 * @param dateEnd 
 */
data class ResponseMapInfoFeaturesPublicTransport(

    @field:JsonProperty("date_start", required = true) val dateStart: java.time.OffsetDateTime,

    @field:JsonProperty("date_end", required = true) val dateEnd: java.time.OffsetDateTime
) {

}

